docs/source/notebooks/AutoEncoders.md

```diff
@@ -73,99 +75,91 @@ print(x_test.shape)
 
 ## 3. Define Keras Model
 
-We will be defining a very simple autencoder. We define _three_ model building methods:
+We will be defining a very simple autencoder. We define _three_ models:
 
-1. One to build a full end-to-end autoencoder.
-2. One to create a model that includes only the encoder portion.
-3. One that creates a model that includes only the decoder portion.
+1. An encoder
+2. A decoder
+3. An autoencoder (which is just a chain of the encoder and decoder)
```

Comment:
These models are of different types. I would add a paragraph explaining that choice.

Reply:
What do you mean by `different types`? Different architectures?

Reply:
The autoencoder is a `keras.Model`. The encoder and decoder are `BaseWrapper`s.

Reply:
I see what you mean now, sorry for being slow.

The intention of this section was to discuss the Keras Model architectures. I edited the text to reflect this. I also added class-attribute type-hints for `encoder_model_` and `decoder_model_` indicating that they are instances of `BaseWrapper`. I hope this helps clarify things.

Reply:
If correct, maybe add a line saying something similar to the following?

> The encoder and decoder are views to the first/last layers of the autoencoder model. They'll be directly used in `predict`/`inverse_transform`/etc, so let's create some SciKeras models with those layers in `{encoder, decoder}_model_`.

Reply:
I added a similar sentence. Thank you for the suggestion.